docs/identity.md

```diff
@@ -0,0 +1,399 @@
+from the [Envoy xDS server in OSM control
+plane](https://github.com/openservicemesh/osm/blob/main/DESIGN.md#1-proxy-control-plane) and allows for dynamically updating which endpoints a Proxy can talk to, enables service certificate rotations and more. Learn more about the [design of
+OSM](../DESIGN.md) which goes into more details on how this works.
+
+#### Envoy Sidecar
+During the Mutating webhook step, the Pod Identity is established using the
+Pod's Service account and namespace.  The Envoy `Proxy Certificate` has the Pod
+Identity encoded in the Common Name (CN) field and DNS Subject Alternative Name (SAN) field of the certificate and is later
+used to confirm Pod Identity and issue certificates for Pod to Pod
+communication. 
+
+The Envoy `Proxy Certificate` CN and SAN DNS name has the following format:
+
+```
+<proxy-UUID>.<kind>.<proxy-identity>.<trust-domain>
+```
+
+- `proxy-uuid` = random unique GUID
+- `kind` = `sidecar` 
+- `proxy-identity` = For Kubernetes this identity is from the pod spec
+  in the form of `<service-account>.<namespace>`
+- `trust-domain` = default is `local.cluster`
+
+An example of this cert can be seen by forwarding the Envoy Admin UI:
+
+```bash
+kubectl port-forward bookwarehouse-78757788bb-zs4hk 15000
+Forwarding from 127.0.0.1:15000 -> 15000
+Forwarding from [::1]:15000 -> 15000
+
+# from another terminal
+curl http://localhost:15000/config_dump | jq -r '.configs[] | select(."@type"=="type.googleapis.com/envoy.admin.v3.SecretsConfigDump") | .dynamic_active_secrets[] | select(.name=="tls_sds").secret.tls_certificate.certificate_chain.inline_bytes' | base64 -d | openssl x509 -noout -text
+
+
+Certificate:
+    Data:
+        Version: 3 (0x2)
+        Serial Number:
+            e2:33:ea:fc:c1:59:75:1c:3e:be:f6:4c:8e:67:4d:ab
+        Signature Algorithm: sha256WithRSAEncryption
+        Issuer: C = US, L = CA, O = Open Service Mesh, CN = osm-ca.openservicemesh.io
+        Validity
+            Not Before: Aug 15 16:37:14 2022 GMT
+            Not After : Aug 12 16:37:14 2032 GMT
+        Subject: O = Open Service Mesh, CN = a00b5103-bcff-40d1-aebc-0fd88cd541a7.sidecar.bookwarehouse.bookwarehouse.cluster.local
+        
+        ...
+
+        X509v3 extensions:
+            ...
+            X509v3 Subject Alternative Name:
+                DNS:a00b5103-bcff-40d1-aebc-0fd88cd541a7.sidecar.bookwarehouse.bookwarehouse.cluster.local
+```
+
+##### Confirming Pod Identity when requesting configuration from xDS
+When a proxy establishes a requests to the OMS Control plane.  The xDS server
+will confirm the envoy side car has a trusted Certificate.  Additionally before
+processing any requests, the xDS service will also gather Pod meta data and
+confirm the Service Account in the presented Certificate matches the service
+Account of the running Pod. 
+
+#### Service to Service communication certificates
+
+When the Proxy requests its certificate for service to service communication,
```

Comment:
"Service to Service" although used in some parts of the code/docs, can be misleading because clients do not need to be services. I'd use the term `client to service` instead of `service to service`.

Reply:
good point, I changed the wording around these several times.